OrangeHRMTests/PageObjects/Modules/TopbarMenu.cs

```diff
@@ -0,0 +1,24 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Modules
+{
+    public static class TopbarMenu
+    {
+        private static MyWebElement ArrowButton = new MyWebElement(By.XPath("//*[contains(@class, 'userdropdown-tab')]//i"));
+        private static MyWebElement AdminPageHeader = new MyWebElement(By.XPath("//div[@class='oxd-topbar-header-title']"));
+
+        private static string TopbarMenuButtonByName = "//nav[contains(@role, 'navigation')]//*[contains(text(), '{0}')]";
```

Comment:
I would better make all locators const and put before elements

Reply:
Fixed